Simple Inventory Management System/Program.cs

```diff
@@ -1,86 +1,102 @@
-﻿using System;
+﻿using Microsoft.Extensions.Configuration;
+using Simple_Inventory_Management_System.Repositories;
 
-namespace Simple_Inventory_Management_System
+namespace Simple_Inventory_Management_System;
+
+class Program
 {
-    class Program
+    static async Task Main(string[] args)
     {
-        static void Main(string[] args)
-        {
-            Inventory inventory = new Inventory();
+        var config = new ConfigurationBuilder()
+            .AddJsonFile(@"appsettings.json", optional: false, reloadOnChange: true)
+            .Build();
 
-            while (true)
-            {
-                Console.WriteLine("\n1. Add Product");
-                Console.WriteLine("2. Display Products");
-                Console.WriteLine("3. Edit Product");
-                Console.WriteLine("4. Delete Product");
-                Console.WriteLine("5. Search for Product");
-                Console.WriteLine("6. Exit");
+        var connectionString = config.GetConnectionString("Default");
 
-                Console.ForegroundColor = ConsoleColor.Yellow;
-                Console.Write("Enter your choice: ");
-                Console.ForegroundColor = ConsoleColor.White;
-                string choice = Console.ReadLine();
+        var productRepository = new ProductRepository(connectionString);
```

Comment:
You used to do dependency injection;I will not ask you to do it here, but this is a reminder to use it in the future